explorer/pages/explore/[slug].tsx

```diff
@@ -0,0 +1,59 @@
+import { GetStaticPaths, GetStaticProps, GetStaticPropsContext } from "next";
+import React from "react";
+import { Page } from "../../app/components";
+import { ListViewModel } from "../../app/models";
+import { ListContainer } from "../../app/project/list";
+import { config } from "app/config";
+import { ParsedUrlQuery } from "querystring";
+import { EMPTY_PAGE, listAll } from "app/project/api";
+import { getCurrentEntity } from "app/hooks/useCurrentEntity";
+
+interface PageUrl extends ParsedUrlQuery {
+  slug: string;
+}
+
+const ListPage: React.FC<ListViewModel> = (props: ListViewModel) => {
```

Comment:
In our other projects, we decided not use `FC` and go with this format instead:

```
export default function ListPage(props: ListProps): JSX.Element {
  return ...
}
```

[Here's](https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html) a related article.

For consistency, can you update this component and throughout?